action.yml

```diff
@@ -32,6 +32,26 @@ inputs:
       for the surest of choices.
     required: false
     default: Changelog
+  versions-dropdownlist-config:
```

Comment:
Try to make this variable name more concise

Reply:
Should `version-tab` be ok?